### PR TITLE
chore: update gitpod configuration

### PR DESCRIPTION
**Pull request by moul**
Can be tested here: https://gitpod.io/new/#https://github.com/gnolang/gno/tree/dev/moul/new-gitpod

<details><summary>Checklists...</summary>

## Contributors Checklist

- [ ] Added new tests, or not needed, or not feasible
- [ ] Provided an example (e.g. screenshot) to aid review or the PR is self-explanatory
- [ ] Updated the official documentation or not needed
- [ ] No breaking changes were made, or a `BREAKING CHANGE: xxx` message was included in the description
- [ ] Added references to related issues and PRs
- [ ] Provided any useful hints for running manual tests
- [ ] Added new benchmarks to [generated graphs](https://gnoland.github.io/benchmarks), if any. More info [here](../.benchmarks/README.md).
</details>

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -1,27 +1,51 @@
...
 tasks:
-  - name: Dev Shell
-    init: >
-      echo "install developer tools" &&
-      (cd misc/devdeps && make install) &&
-      echo "download dependencies" &&
-      go mod download &&
-      go install ./gnovm/cmd/gno
+  - name: Gno Shell
+    before: cd ./examples/
```
Comment (zivkovicmilos) [inline .gitpod.yml:5]: Just curious, why cd into `examples`, only to cd out in the `init` command?

File: .gitpod.yml [hunk 2]
```diff
@@ -1,27 +1,51 @@
...
-    init: go install ./gno.land/cmd/gnoweb
+    before: cd ./gno.land/
+    init: go install ./cmd/gnoweb
     command: gnoweb --bind=0.0.0.0:8888
 
-  - name: Gnoland Node
-    init: go install ./gno.land/cmd/gnoland
-    command: gnoland start
+  #- name: faucet
+  #  ...
```
Comment (zivkovicmilos) [inline .gitpod.yml:30]: Leftover?

**Review by zivkovicmilos — APPROVED** [review]
Looks good 💯 

I've tried it out on the gitpod link you've provided ✅

File: .gitpod.yml (hunk 2 above)
Comment (moul) [reply .gitpod.yml:30]: suggestion for improvements :)

File: .gitpod.yml (hunk 1 above)
Comment (moul) [reply .gitpod.yml:5]: before is always applied, for `init` AND for `command`. when using cached images on gitpod, the `init` directive is skipped.

`before` needs to be global, and then `init` is an exception.

**Pull request merged**